src/Compilers/CSharp/Portable/Symbols/Wrapped/WrappedTypeParameterSymbol.cs

```diff
@@ -73,6 +73,14 @@ public override bool HasReferenceTypeConstraint
             }
         }
 
+        public override bool HasUnmanagedTypeConstraint
```

Comment:
>public override bool HasUnmanagedTypeConstraint [](start = 8, length = 47)

Do we have tests covering all subclasses of the enclosing class?

Reply:
can you please clarify? which scenario do you see uncovered by this property?

---
In reply to: [172419029](https://github.com/dotnet/roslyn/pull/24817#discussion_r172419029) [](ancestors = 172419029)

Reply:
> can you please clarify? which scenario do you see uncovered by this property?

I asked the question whether you covered all leaf classes derived from this one in tests. Did you?

---
In reply to: [175999068](https://github.com/dotnet/roslyn/pull/24817#discussion_r175999068) [](ancestors = 175999068,172419029)

Reply:
Added tests for retargeting and non-synthesized type parameters. We already have tests for synthesized ones.

---
In reply to: [176269706](https://github.com/dotnet/roslyn/pull/24817#discussion_r176269706) [](ancestors = 176269706,175999068,172419029)

Reply:
> Added tests for retargeting and non-synthesized type parameters. We already have tests for synthesized ones.

What about SubstitutedTypeParameterSymbol?

---
In reply to: [177881475](https://github.com/dotnet/roslyn/pull/24817#discussion_r177881475) [](ancestors = 177881475,176269706,175999068,172419029)